src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -310,6 +310,23 @@ struct EvolutionMetavars {
                      volume_dim, ::Frame::Inertial>,
```

Comment:
Can you split this into 2 commits, the first that adds the tag and its test, the second that modifies the GH executables?

Also, can you make commit names be active, "Add X" rather than "Adding X"?